### DISCUSSION TITLE
exporting GDSII file

### DISCUSSION THREAD

**Discussion by Nikolaos-Matthaiakakis** [NanoComp/meep#2124] [Q&A] [answered]
Is there an easy way to export a meep geometry to a GDSII file? For example this would be useful for metasurface simulations since the optimised structure can then be exported for fabrication.

**Comment by oskooi** [comment, marked as answer]
To create GDS files, check out the two packages https://github.com/amccaugh/phidl and https://github.com/gdsfactory/gdsfactory.

**Comment by Nikolaos-Matthaiakakis** [comment]
Thank you, I will check them out.
